tee-worker/omni-executor/parentchain/listener/src/lib.rs

```diff
@@ -137,22 +138,18 @@ where
 
 	perform_attestation(client_factory, signer, &transaction_signer).await?;
 
-	let intent_event_handler = IntentEventHandler::new(
+	let account_store_storage = Arc::new(AccountStoreStorage::new());
```

Comment:
It's initiated from the disk, which means we can't skip any parentchain blocks.

Shall we combine it with the download of AccountStore at the program start?

Reply:
Yes, that's the plan. We already have an [issue](https://linear.app/litentry/issue/P-1251/download-all-accountstores-if-they-dont-exists-when-starting-the-omni) for it